doc/sphinx/addendum/type-classes.rst

```diff
@@ -3,16 +3,16 @@
 Typeclasses
 ===========
 
-This chapter presents a quick reference of the commands related to type
-classes. For an actual introduction to typeclasses, there is a
-description of the system :cite:`sozeau08` and the literature on type
-classes in Haskell which also applies.
+This chapter presents a quick reference of the commands related to typeclasses.
+For an actual introduction to typeclasses, there is a description of the system
+:cite:`sozeau08` and the literature on type classes in Haskell which also
+applies.
```

Comment:
```suggestion
This chapter describes the commands related to typeclasses.
Typeclasses are described in more detail in :cite:`sozeau08`.
The Haskell type class literature is also relevant.
```

"This chapter ..." is very weak.  It would be better to start with a sentence describing the benefit to the user, if you're willing.

relevent -> helpful?

Reply:
Yes, what should we write instead.